Userland/Applications/Browser/Tab.cpp

```diff
@@ -38,8 +39,15 @@
 
 namespace Browser {
 
+static String s_search_engine_format = "{}";
+
 URL url_from_user_input(const String& input)
 {
+    if (!input.is_empty() && input[0] == '?' && s_search_engine_format != "{}"sv) {
+        dbgln("Using search engine for {}", input);
+        return URL(String::formatted(s_search_engine_format, input));
```

Comment:
You still need to remove the leading `?`, no?

Reply:
Fixed.